workshops/test/test_person.py

```diff
@@ -183,7 +187,8 @@ def test_person_award_badge(self):
 
         rv = self.client.post(url, data=values)
         assert rv.status_code == 302, \
-            'After awarding a badge we should be redirected to the same page, got {} instead'.format(rv.status_code)
+            'After awarding a badge we should be redirected to the same ' \
+            'page, got {} instead'.format(rv.status_code)
```

Comment:
Nitpicking: you can, and we do that in some places, surround the `text.format()` part in parentheses and then you don't need backslashes.